app/controllers/api/v1/doctors_controller.rb

```diff
@@ -3,9 +3,9 @@ class Api::V1::DoctorsController < ApplicationController
 
   # GET /doctors
   def index
-    @doctors = Doctor.all
+    @doctors = Doctor.all.joins(:photo_attachment)
 
-    render json: @doctors
+    render json: @doctors.map { |doctor| doctor.as_json.merge({ photo_url: url_for(doctor.photo) }) }
```

Comment:
- Could you please look at the screenshot attached below .The form data is not being added to the database.

![Screenshot from 2023-08-29 14-36-49](https://github.com/sumon766/backend-doctor-appointment/assets/86189460/5203d873-82b3-4bd9-97b1-a28f34153f1d)